src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseFVMKernels.hpp

```diff
@@ -37,12 +37,14 @@
 #include "physicsSolvers/fluidFlow/SinglePhaseBaseFields.hpp"
 #include "physicsSolvers/fluidFlow/SinglePhaseBaseKernels.hpp"
 #include "physicsSolvers/fluidFlow/StencilAccessors.hpp"
+#include "physicsSolvers/fluidFlow/FluxKernelsHelper.hpp"
 
 namespace geos
 {
 
 namespace singlePhaseFVMKernels
 {
+using namespace fluxKernelsHelper;
```

Comment:
please don't add `using namespace` inside of headers.

Reply:
removed it